wie-geht-studium/mafin.tex

```diff
@@ -0,0 +1,12 @@
+\subsection{Mathematik für Informatiker}
+\label{mafin}
+
+Die Mathematikausbildung für InformatikerInnen ist etwas kompliziert. Ihr habt die Auswahl zwischen den Vorlesungen Lineare Algebra I und Analysis I oder den Vorlesungen Mathematik für Informatiker I und II. Ihr entscheidet euch also bereits im ersten Semester für eine der beiden Varianten. Deshalb solltet ihr euch schon jetzt mit diesem Thema auseinandersetzen.\\
+
+Was spricht für Mathematik für Informatiker (MafIn)?\\
+Die Vorlesung ist extra auf InformatikerInnen zugeschnitten und versucht euch den Stoff besser verständlich näher zu bringen. Die Mathematik für Informatiker I orientiert sich dabei an den grundlegenden Inhalten der Linearen Algebra I, die Mathematik für Informatiker II an denen der Analysis I. Grundsätzlich ist die Vorlesung recht solide konzipiert und kann euch die wichtigen Mathematik-Grundlagen vermitteln. Leider wurde diese Vorlesung in den letzten Jahren immer vom selben Dozenten gehalten, mit dem es einige Probleme in der Vorlesung und dem zugehörigen Übungsbetrieb gab. Ihr solltet euch aber selbst ein Bild von der Vorlesung machen, ob sie euch zusagt oder nicht. Lasst uns gerne Feedback zur Vorlesung zu kommen\footnote{Feedback könnt ihr uns am besten per Mail, oder bei der Informatik-Vollversammlung zukommen lassen.}.\\
+
+Was spricht für Lineare Algebra I (LA I)\footnote{Details zur LA I siehe Seite \pageref{la1}} und Analysis I (Ana I)\footnote{Details zur Ana I siehe Seite \pageref{ana1}}?\\
+Die Vorlesungen sind für MathematikerInnen konzipierte Veranstaltungen. Diese bieten dadurch eine deutlich präzisere Herangehensweise an und Formulierung der Definitionen, Sätze und insbesondere Beweise. Ihr bekommt dadurch eine deutlich solidere Mathematikausbildung die ihr in manchen Bereichen der Informatik auch dringend benötigt. Die LA I und Ana I bedeutet für euch aber auch meist mehr Arbeit und Zeitaufwand, da ihr auch eine deutlich intensivere und vertiefendere Ausbildung bekommt.\\
```

Comment:
Den zweiten Satz könnte man vielleicht noch etwas besser formulieren. Mir selbst fällt aber gerade keine gute Möglichkeit ein.
3. Satz fehlt ein Komma zwischen "[…]solidere Mathematikausbildung , die ihr in[…]"